core/rs/core/src/lib.rs

```diff
@@ -471,6 +473,40 @@ pub extern "C" fn sqlite3_crsqlcore_init(
         return null_mut();
     }
 
+    let rc = db
+        .create_function_v2(
+            "crsql_config_set",
+            2,
+            sqlite::UTF8,
+            Some(ext_data as *mut c_void),
+            Some(crsql_config_set),
+            None,
+            None,
+            None,
+        )
+        .unwrap_or(sqlite::ResultCode::ERROR);
+    if rc != ResultCode::OK {
+        unsafe { crsql_freeExtData(ext_data) };
+        return null_mut();
+    }
+
+    let rc = db
+        .create_function_v2(
+            "crsql_config_get",
+            1,
+            sqlite::UTF8,
```

Comment:
can you add the `sqlite::INNOCUOUS` and `sqlite::DETERMINISTIC`?

https://www.sqlite.org/c3ref/c_deterministic.html